doc/Command-Reference.md

```diff
@@ -9517,6 +9517,42 @@ This command displays brief information about all the vnets configured in the de
   Vnet_3000    tunnel1          3000  Vnet_2000,Vnet4000
   ```
 
+**show vnet endpoint <ip/ipv6>**
+
+This command displays the list or vxlan tunnel endpoints and their status. In addition it also shows the number of prefixes associated with each endpoints in the tunnels. If an IP address of an endpoint is provided, this command also shows the associated prefixes a well.
+
+- Usage:
+
+  ```
+  show vnet endpoint <ipv4_address/ipv6_address>
```

Comment:
Can we also print the "`endpoint_monitor`" ip so we have a mapping? BFD session just have` endpoint_monitor'`

Reply:
ok

Reply:
done